packages/react-native/src/handlers/android/AndroidUnhandledExceptionHandler.ts

```diff
@@ -0,0 +1,42 @@
+import { BacktraceReport } from '@backtrace-labs/sdk-core';
+import { NativeModules } from 'react-native';
+import { BacktraceClient } from '../../BacktraceClient';
+import { DebuggerHelper } from '../../common/DebuggerHelper';
+import { UnhandledExceptionHandler } from '../UnhandledExceptionHandler';
+export class AndroidUnhandledExceptionHandler extends UnhandledExceptionHandler {
+    private readonly _unhandledExceptionHandler = NativeModules.BacktraceAndroidBackgroundUnhandledExceptionHandler;
+    public captureManagedErrors(client: BacktraceClient) {
+        super.captureManagedErrors(client);
+        if (!this._unhandledExceptionHandler) {
+            return;
+        }
+        if (DebuggerHelper.isConnected()) {
+            return;
+        }
+
+        this._unhandledExceptionHandler.start(async (classifier: string, message: string, stackTrace: string) => {
```

Comment:
Can we skip `async/await`? Remember what it does to stack frames.

Reply:
I forgot! Yes